testing/jupyterlab/playwright.config.ts

```diff
@@ -39,7 +39,21 @@ const config: PlaywrightTestConfig = {
   workers: process.env.CI ? 1 : undefined,
 
   /* Reporter to use. See https://playwright.dev/docs/test-reporters */
-  reporter: "html",
+  reporter: [
+    [process.env.CI ? "github" : "list"],
+    [
+      "./markdown-reporter.ts",
+      {
+        outputFile: "test-results/jupyterlab-a11y-regression-test-results.md",
+        /* The URL to the test info pages directory */
+        testInfoPagesBaseURL: process.env.CI
+          ? `${process.env.GITHUB_SERVER_URL}/${process.env.GITHUB_REPOSITORY}/blob/${process.env.GITHUB_SHA}/testing/jupyterlab/test-info-pages`
+          : "https://github.com/Quansight-Labs/jupyter-a11y-testing/blob/main/testing/jupyterlab/test-info-pages",
```

Comment:
Is this so that it always points to the file within the PR?

Reply:
Yes, line 50 should allow the GitHub Action to run against a PR that adds a new test + test info page (you should never add a test without adding a test info page). It should allow it to create a markdown report in the GitHub workflow run summary in which the link to the test info page isn't broken. The link to the test info page in the summary should take you to the info page in the working branch of the PR.

Reply:
I'll add a comment explaining the purpose of this line.